ruby/rails/security/brakeman/check-reverse-tabnabbing.yaml

```diff
@@ -0,0 +1,57 @@
+rules:
+- id: check-reverse-tabnabbing
+  mode: search
+  paths:
+    include:
```

Comment:
Some of the other Ruby rules only check *.erb. Does this path run the risk of missing HTML in files without the .html.erb extension?

Reply:
this rule only works on Embedded Ruby (`erb`) templates - it will be nonsensical otherwise.  The Rails engine expects templating files to have these extensions.